### ISSUE TITLE
Support "deterministic policy option" when running policy in AnyLogic

### ISSUE THREAD

**Issue by thetwotravelers** [PathmindAI/nativerl#102]
We have trained a policy for the Order Sequencing model that most often takes the optimal sequence of actions, leading to the minimum processing time of 6766 seconds. However, in some simulation runs it takes a different set of actions, leading to processing times as high as 9142 seconds.

The model itself is deterministic, so all the variance in results in the "Time to Completion" below are due to the stochastic nature of the PPO generated policy.

![os_policy_behaviour](https://user-images.githubusercontent.com/56656491/87825523-28fce500-c82c-11ea-9b6a-a7953b788f11.png)

NOTE: This policy was trained in the webapp and was likely cut short. Longer training may collapse the behaviour further to become increasingly close to deterministic. However, it would greatly lessen the training time if we could manually collapse it by freezing it into a deterministic mapping.

The stochastic nature of the policy is useful in training for exploration, but loses it's utility when the policy is being deployed in a deterministic environment.

Therefore I'm suggesting we add a feature to the PathmindHelper which toggles a parameter in RLlibPolicyHelper.java that changes the way actions are generated from the TensorFlow model.

It essentially requires finding the index of the largest action logit, something like the following:
````
policy = agent.get_policy()
action, _, info = policy.compute_single_action(observation)
sampled_action = action # stochastic
greedy_action = info['logits'].argmax()  # deterministic
````

In RLlibPolicyHelper.java, it appears the action logits are available to access and use to choose the greedy action:

https://github.com/SkymindIO/nativerl/blob/155b90d148bba6f689c8867af13068edc8824dbe/nativerl/src/main/java/ai/skymind/nativerl/RLlibPolicyHelper.java#L71

@saudet Does the above seem doable?

I believe if this option becomes available, then the current Order Sequencing policy would change from a suboptimal stochastic policy to an optimal deterministic policy - without any additional training. This may be a useful feature for other deterministic models and also to give customers a sense of predictability when a policy is in deployment (policy serving).

Model + Policy for reproduction:
[Order_sequencing_TO_07172020_HeuristicIntegration_brett.zip](https://github.com/SkymindIO/nativerl/files/4939993/Order_sequencing_TO_07172020_HeuristicIntegration_brett.zip)
